ReleaseNotes/eng/Altibase_7_3_0_0_1_Release_Notes.md

```diff
@@ -0,0 +1,857 @@
+
+##### Sequence Restart Statement
+
+Altibase 7.3 supports the restart_clause with ALTER SEQUENCE statement. Refer to [**SQL Reference Manual** - restart_clause](https://github.com/ALTIBASE/Documents/blob/master/Manuals/Altibase_7.3/eng/SQL%20Reference.md#restart_clause).
+
+</br>
+
+#### 2.1.6 Functionality Improvement - Spatial SQL
+
+##### SRID Interface
+
+A Spatial Reference Identifier(SRID) is an identifier used to distinguish spatial objects, represented by an integer within a 4-byte range, and can be applied to GEOMETRY columns. Users can specify the SRID when creating a table and also alter the SRID using the ALTER TABLE statement.
+
+New ways to represent GEOMETRY data types:
+
+- Extended Well-Known Text (EWKT) format: It includes SRID information in the WKT format to represent spatial objects.
+- Extended Well-Known Binary (EWKB) format: It includes SRID information in the WKB format to represent spatial objects.
+
+##### Spatial Functions
+
+New functions have been introduced as follows.
+
+* ASEWKT
+* ASEWKB
+* GEOMFROMEWKT
+* GEOMFROMEWKB
+* SETSRID
+* SRID
+* ST_Collect
+* ST_IsCollection
+* ST_LinestringFromWKB
+* ST_MakeEnvelope
+* ST_MakeLine
+* ST_MakePoint
+* ST_Point
+* ST_PolygonFromText
+* ST_Transform
+
+##### Spatial Object Creation Functions
+
+A new function has been introduced as follows.
+
+* ACSGetGeometrySRID
+
+#### 2.1.7 Functionality Improvement - Replication
+
+##### DDL Synchronization
+
+DDL Synchronization is now possible through replication. To use this feature, users must set the REPLICATION_DDL_SYNC and REPLICATION_DDL_ENABLE properties on each node to 1. Also, the REPLICATION_DDL_ENABLE_LEVEL property of each node must be set the same.
+
+To use DDL Synchronization, the following constraints must be verified:
+
+- Replication must be operational on the nodes where DDL Synchronization is performed.
+- The table names on both local and remote nodes for DDL Synchronization must match.
+- The table partition names on both local and remote nodes for DDL Synchronization must match.
+- The owner names of tables on both local and remote nodes for DDL Synchronization must match.
+- Only one node can perform DDL Synchronization at a time.
+- The values of the REPLICATION_DDL_ENABLE and REPLICATION_DDL_ENABLE_LEVEL properties must be identical for each node.
+- The Altibase Server version(5 digits) must be the same.
+- DDL Synchronization is not permitted when using the Propagation option.
+
+#####  RECEIVE_ONLY Option
+
+A new option, RECEIVE_ONLY, has been introduced to prevent the transmission of transaction logs for changing data to other nodes. When creating replication with the RECEIVE_ONLY option, transaction logs do not affect other nodes because they are not read. Therefore, any replication issues such as network failures do not affect the system.
```

Comment:
```suggestion
A new option, RECEIVE_ONLY, has been introduced to prevent the transmission of transaction logs for changing data to other nodes. If users create a replication with the RECEIVE_ONLY option, the local server does not read the transaction logs of the remote server. Therefore, any replication issues such as network failures do not affect the local server system.
```

Reply:
If users create a replication with the RECEIVE_ONLY option -> If users create a replication with the RECEIVE_ONLY option on remote server, 가 맞을것 같네요